data_pipeline.py

```diff
@@ -2,4 +2,6 @@
 
 # sqlalchemy changes 
 
-#333333333333333333333333333333
\ No newline at end of file
+#333333333333333333333333333333
```

Comment:
@darpan810 :: this is not the correct way to right the code

Reply:
yes i will make it correct ,Thanks